index.jsx

```diff
@@ -137,11 +137,19 @@ module.exports = function Dimensions ({
         this.getWindow().removeEventListener('resize', this.onResize)
       }
 
+      getWrappedInstance = () => this.refs.wrappedInstance;
```

Comment:
@mjrussell any reason that this needs to be bound? react-redux just uses a standard instance method e.g. `getWrappedInstance() { return this.refs.wrappedInstance }`